release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -770,291 +770,417 @@ Some features in this release are currently in Technology Preview. These experim
+
+.Monitoring Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |OCP 4.10 |OCP 4.11 | OCP 4.12
 
 |Alert routing for user-defined projects monitoring
-|TP
-|GA
-|GA
+|Technology Preview
+|General Availability
+|General Availability
 
-|Disconnected mirroring with the oc-mirror CLI plug-in
-|TP
-|GA
-|GA
+|Alerting rules based on platform monitoring metrics
+|Not Available 
+|Technology Preview
+|Technology Preview
 
-|Mount shared entitlements in BuildConfigs in RHEL
-|TP
-|TP
-|TP
+|====
+
+[discrete]
+=== {rh-openstack-first} Technology Preview features
+
+.{rh-openstack} Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |OCP 4.10 |OCP 4.11 | OCP 4.12
 
 |Support for {rh-openstack} DCN
-|TP
-|TP
-|TP
+|Technology Preview
+|Technology Preview
+|Technology Preview
 
 |Support for external cloud providers for clusters on {rh-openstack}
-|TP
-|TP
-|TP
+|Technology Preview
+|Technology Preview
+|Technology Preview
 
 |OVS hardware offloading for clusters on {rh-openstack}
-|TP
-|GA
-|GA
+|Technology Preview
+|General Availability
+|General Availability
 
-|External DNS Operator
-|TP
-|TP
-|TP
+|====
 
-|Alerting rules based on platform monitoring metrics
-|-
-|TP
-|TP
+[discrete]
+=== Architecture Technology Preview features
 
-|AWS Load Balancer Operator
-|-
-|TP
-|TP
+.Architecture Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |OCP 4.10 |OCP 4.11 | OCP 4.12
 
-|Node Observability Operator
-|-
-|TP
-|TP
+|Hosted control planes for {product-title}
+|Not Available
+|Technology Preview
+|Technology Preview
 
-|Java-based Operator
-|-
-|TP
-|TP
+|====
 
-|Hosted control planes for {product-title}
-|-
-|TP
-|TP
+[discrete]
+=== Machine management Technology Preview features
+
+.Machine management Technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |OCP 4.10 |OCP 4.11 | OCP 4.12
 
 |Managing machines with the Cluster API
-|-
-|TP
-|TP
+|Not Available
+|Technology Preview
+|Technology Preview
 
-|Linux Control Group version 2 (cgroup v2)
-|-
-|-
-|TP
+|Cron job time zones
+|Not Available
+|Not Available
+|Technology Preview
 
-|crun container runtime
-|-
-|-
-|TP
+|Cloud controller manager for Alibaba Cloud
+|Technology Preview
+|Technology Preview
+|Technology Preview
 
-|Cron job time zones
-|-
-|-
-|TP
+|Cloud controller manager for Amazon Web Services
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Cloud controller manager for Google Cloud Platform
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Cloud controller manager for Microsoft Azure
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Cloud controller manager for {rh-openstack-first}
+|Technology Preview
+|Technology Preview
+|Technology Preview
+
+|Cloud controller manager for VMware vSphere
+|Technology Preview
+|Technology Preview
+|Technology Preview
 
 |Custom Metrics Autoscaler Operator
```

Comment:
This one got missed for expanding the status names, but also doesn't belong in this section. I think it goes in Networking based on the [feature blurb](https://52611--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-nw-ingress-autoscaling) for it.

Reply:
Thank you! 😄